tests/test_items.py

```diff
@@ -0,0 +1,117 @@
+"""
+Test suite for ItemHandler and ItemListHandler
+"""
+
+import json
+
+from peewee import SqliteDatabase
+import http.client as client
+
+from app import app
+from models import Item
+
+TEST_ITEM = {
+    'name': 'mario',
+    'price': 20.20,
+    'description': 'svariati mariii'
+}
+TEST_ITEM2 = {
+    'name': 'GINO',
+    'price': 30.20,
+    'description': 'svariati GINIIIII'
+}
+TEST_ITEM_WRONG = {
+    'name': '',
+    'price': 30.20,
+    'description': 'svariati GINIIIII'
+}
+TEST_ITEM_PRECISION = {
+    'name': 'Anna Pannocchia',
+    'price': 30.222222,
+    'description': 'lorem ipsum'
+}
+
+
+class TestItems:
+    @classmethod
+    def setup_class(cls):
+        Item._meta.database = SqliteDatabase(':memory:')
+        Item.create_table()
+        cls.app = app.test_client()
+
+    def setup_method(self):
+        Item.delete().execute()
+
+    def test_post_item__success(self):
+        resp = self.app.post('/items/', data=json.dumps(TEST_ITEM),
+                             content_type='application/json')
+        assert resp.status_code == client.CREATED
+        assert len(Item.select()) == 1
+        assert Item.select()[0].json() == TEST_ITEM
+        assert Item.select()[0].id == json.loads(resp.data)['item_id']
```

Comment:
In questo test stai facendo 3 volte la select(), che vuol dire rallentare senza particolari ragione l'esecuzione dei test. Come regola generale gli unit test devono essere veloci da eseguire (altrimenti essendo molti poi diventa non conveniente eseguirli spesso)